src/app/components/data-entry-grid/index.html

```diff
@@ -102,6 +102,12 @@ <h3>Guidelines</h3>
       >
       </sky-demo-page-module-info>
 
+      <sky-docs-demo-page-module-info>
+        <p>
+          In addition to the SKY UX package, you need to install the <sky-code>ag-grid-angular</sky-code> and <sky-code>ag-grid-community</sky-code> packages. After installation, you need to make any exports from <sky-code>SkyAgGridModule</sky-code> available to your SPA by adding the module to the <sky-code>exports</sky-code> section of your SPA's main module. If you are using the SKY UX SDK, add <sky-code>SkyAgGridModule</sky-code> as an export in your <sky-code>app-sky.module.ts</sky-code> file. Next, in the <sky-code>app-extras.module.ts</sky-code> file, add <sky-code>AgGridModule.withComponents([])</sky-code> to the <sky-code>imports</sky-code> section and <sky-code>AgGridModule</sky-code> to the <sky-code>exports</sky-code> section. If you have defined your own cell editors or renderers, add them to the list passed to <a href="https://www.ag-grid.com/angular-more-details/#angular-components"><sky-code>withComponents()</sky-code></a>.
```

Comment:
In the opening clause, should "SKY UX package" be "`@skyux/ag-grid` package" since we use code snippets for the other package names in this sentence? And further on in the first sentence, should we link to the ag-grid-angular and ag-grid-community packages? Seems like that's essential info. @Blackbaud-ErikaMcVey, what do you think?

@Blackbaud-DeniseCisneros, do you think we should start the third sentence with "For example, if you are..."? (I know the wording you have in place is my suggested wording, so sorry to offer an edit on the edit I already suggested. ;-)

And in the last sentence, the verb tenses seem off to me. We use past tense, but this is the first step in implementing data entry grid, so past tense seems wrong to me. What do you think of tweaking this to "If you define your own cell editors or renderers, add them to the list you pass to `withComponents()`."